plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy_test.go

```diff
@@ -136,3 +138,28 @@ func TestEditViewRelationship(t *testing.T) {
 		t.Errorf("view is missing rules for: %#v\nIf these are escalating powers, add them to the list.  Otherwise, add them to the view role.", miss)
 	}
 }
+
+func TestBootstrapLabel(t *testing.T) {
+	roles := bootstrappolicy.ClusterRoles()
```

Comment:
There are roles and bindings created for controllers.  Different file, same package.